backend/views/views.py

```diff
@@ -0,0 +1,115 @@
+from datetime import datetime
+from flask import flash, render_template, redirect, request, session, url_for
+from manage import app, db
+from forms.forms import LoginForm, UserForm
+from models.users import Role, User
+from models.issues import Attachment, Category, Issue, IssueHistory, Status
+
+
+@app.route('/')
+@app.route('/index')
+def index():
+    return render_template('index.html')
+
+
+@app.route('/admin')
+def admin():
+    if 'user_id' in session and session['role_id'] == 1:
+        return render_template('admin_page.html')
+    else:
+        flash('Dont have access ...')
+        return redirect(url_for('index'))
+
+@app.route('/user_page')
+def user_page():
+    users = db.session.query(User, Role).filter(
+        User.role_id == Role.id).order_by(User.id).all()
+    return render_template('user_page.html', users=users)
+
+
+@app.route("/user_modify", methods=['GET', 'POST'])
+def user_modify():
+    form = UserForm(request.form)
+
+    if 'id' in request.args:
+        user = db.session.query(User).get(request.args.get('id'))
+        form = UserForm(obj=user)
+
+    if request.method == "GET":
+        return render_template('user_modify.html', form=form)
+
+    elif request.method == "POST":
+
+        if form.validate_on_submit():
+            if form.id.data:
+                user = db.session.query(User).get(form.id.data)
+                user.name = form.name.data
+                user.alias = form.alias.data
+                user.email = form.email.data
+                user.role_id = form.role_id.data
+
+                if form.delete_date.data:
+                    user.delete_date = form.delete_date.data
+                else:
+                    user.delete_date = None
+
+                db.session.commit()
+                flash("user modified")
+            else:
+                newuser = User(name=form.name.data,
+                               alias=form.alias.data,
+                               email=form.email.data,
+                               password=None,
+                               role_id=form.role_id.data,
+                               avatar=None,
+                               delete_date=None)
```

Comment:
You can create  newuser with **kwargs

Reply:
changed to **kwargs, but it not very simple, because we need to delete 3 values from form.data